ext/oci8/oci8_interface.c

```diff
@@ -629,27 +626,18 @@ PHP_FUNCTION(oci_lob_erase)
 	php_oci_descriptor *descriptor;
 	ub4 bytes_erased;
 	zend_long offset = -1, length = -1;
+	zend_bool offset_is_null = 1, length_is_null = 1;
 
-	if (zend_parse_method_parameters(ZEND_NUM_ARGS(), getThis(), "O|ll", &z_descriptor, oci_lob_class_entry_ptr, &offset, &length) == FAILURE) {
+	if (zend_parse_method_parameters(ZEND_NUM_ARGS(), getThis(), "O|l!l!", &z_descriptor, oci_lob_class_entry_ptr, &offset, &offset_is_null, &length, &length_is_null) == FAILURE) {
```

Comment:
According to https://github.com/php/php-src/blob/a249980006477ee48b44fa26a3b3d48c4c24633c/Zend/zend_API.h#L1851-L1853 the `offset`/`length` will get set to zero for a null argument and will not pick the default of `-1`. So I think this logic will be incorrect.

Reply:
Keeping in mind that oci8 cannot be tested in CI, I would suggest to split this part off into a separate PR, to avoid blocking it.

Reply:
Makes sense! And the good news is that I managed to find out how to revert changes to a path in a previous commit:

> Revert (reset) a single file to a specific revision:
$ git checkout <commit_hash> -- <file>

So I didn't have to force push. :)